### ISSUE TITLE
Jaydata 1.4alpha - include is not working with expansiondepth >= 2 with oData v4

### ISSUE THREAD

**Issue by Malyngo** [jaystack/jaydata#214]
When I use a query with an expansion depth that is >= 2, this is not working with oData v4.
The reason is that once again, they changed how this is working in v4.

In v3, an example for such an request is:

```
http://localhost/odata/Categories(1)?$expand=Products/Supplier
```

But in v4, this should look like this:

```
http://localhost/odata/Categories(1)?$expand=Products($expand=Supplier)
```

**Comment by robesz** [comment]
This will be resolved in the next release - Nov-Dec

**Comment by Malyngo** [comment]
That is great to hear!
As this is kinda related, I guess I maybe can ask it here: 
Will you also support $level in the next release?
So I can create queries like these:

```
http://localhost/odata/Categories(1)?$expand=Products($level=2)
```

which would return something like
Category
-- Product
----- Product
-- Product

**Comment by Malyngo** [comment]
fixed in version 1.5 :)
